### PR TITLE
docs: minor fixes

### PR DESCRIPTION
**Pull request by ryoppippi**
Minor fixes!

### CODE REVIEW

File: website/pages/docs/setup.mdx
Comment (samchon) [inline website/pages/docs/setup.mdx:1]: Isn't it options of typia?

**Review by samchon — COMMENTED** [review]
`hono.mdx` is okay, but there's something to check on `setup.mdx`.

File: website/pages/docs/setup.mdx
Comment (ryoppippi) [reply website/pages/docs/setup.mdx:1]: No options of Typia is inside `Options`
https://jsr.io/@ryoppippi/unplugin-typia/doc/~/Options

Comment (samchon) [reply website/pages/docs/setup.mdx:1]: Good document.

**Pull request merged**